pkg/providers/amifamily/bootstrap/bottlerocket.go

```diff
@@ -77,7 +77,8 @@ func (b Bottlerocket) Script() (string, error) {
 
 	s.Settings.Kubernetes.NodeTaints = map[string][]string{}
 	for _, taint := range b.Taints {
-		s.Settings.Kubernetes.NodeTaints[taint.Key] = append(s.Settings.Kubernetes.NodeTaints[taint.Key], fmt.Sprintf("%s:%s", taint.Value, taint.Effect))
+		s.Settings.Kubernetes.NodeTaints[taint.Key] = append(s.Settings.Kubernetes.NodeTaints[taint.Key], fmt.Sprintf("%s%s",
```

Comment:
interesting, so our behavior was wrong before? We would be formatted incorrectly if there was no value?

Reply:
Correct, without the value in the old formatting we end up with `[':NoExecute']`

Reply:
Interesting. Wonder if anyone hit this. Maybe worth calling out as a fix in the description?